en/publications.md

```diff
@@ -7,24 +7,17 @@ permalink: /en/publications/
 share: false
 version: 1
 ---
-{%- if jekyll.environment == 'future' -%}
-- [Scaling book][]: a guide to effective practices and techniques
-  that engineers interacting with the Bitcoin block chain can adopt
-  today.
-
 - [Newsletters][]: a weekly summary of news about Bitcion and LN
   development.
 
 - [Blog posts][]: Occasional updates and reference material
   from the Optech team.
 
-## Latest publications
+- [Podcast Episodes][]: Audio discussions of our newsletters.
 
-{% else %}
 {:.center}
 Recent publications from our [blog posts][] and [newsletters][].
```

Comment:
Now that we have the list, I think this "Recent publications..." line is redundant and should be removed.